src/meson.build

```diff
@@ -718,6 +718,7 @@ if get_option('tests').enabled()
 
   env = environment()
   env.set('srcdir', meson.current_source_dir())
+  env.set('top_srcdir', meson.source_root())
```

Comment:
This probably should be added to `TESTS_ENVIRONMENT` in `Makefile.am` as well.

Reply:
Done. Out of curiosity, is there any plan to drop autotools in favor of Meson in the future?

Reply:
There is an vague plan to drop it at some point in the future and I think all HarfBuzz developers are using meson now, but supporting autotools is low effort and we are familiar with it (unlike, say, CMake), so we are probably going to keep it around for a little while yet. After all, we are still keeping CMake support even though we are not actively maintaining it.